spring-kafka-docs/src/main/antora/modules/ROOT/pages/whats-new.adoc

```diff
@@ -40,4 +40,9 @@ See xref:retrytopic/features.adoc#exc-based-custom-dlt-routing[Routing of messag
 === After Rollback Processing
 
 A new `AfterRollbackProcessor` API `processBatch` is provided.
-See xref:kafka/annotation-error-handling.adoc#after-rollback[After-rollback Processor] for more information.
\ No newline at end of file
+See xref:kafka/annotation-error-handling.adoc#after-rollback[After-rollback Processor] for more information.
+
+[[x32-retry-topic]]
+=== Change @RetryableTopic SameIntervalTopicReuseStrategy default value
+Change `@RetryableTopic` property SameIntervalTopicReuseStrategy default value to SINGLE_TOPIC.
```

Comment:
Code formatting needed for `SameIntervalTopicReuseStrategy` and `SINGLE_TOPIC.`

Reply:
thanks, fix it.